style.css

```diff
@@ -66,6 +67,87 @@ form input {
     fill: var(--red);
 }
 
+/* main card */
+
+main {
+    display: flex;
```

Comment:
I think we can use the `flex` class directly on the `<main>` element and remove this line.